app/connectors/cache/UserAnswersCacheConnector.scala

```diff
@@ -55,12 +56,20 @@ class UserAnswersCacheConnectorImpl @Inject()(
       }
   }
 
-  override def save(id: String, value: JsValue)(implicit
-                                                ec: ExecutionContext,
-                                                hc: HeaderCarrier
+  override def save(id: String, value: JsValue)(implicit ec: ExecutionContext, hc: HeaderCarrier): Future[JsValue] = {
+    save(id, value, url)
+  }
+
+  override def setLock(id: String, value: JsValue)(implicit ec: ExecutionContext, hc: HeaderCarrier): Future[JsValue] = {
```

Comment:
As this is doing more than setting the lock I'd call it something like saveWithLock or saveAndLock